packages/react-components/src/Status/index.tsx

```diff
@@ -144,8 +145,24 @@ function Status ({ className, stqueue = [], txqueue = [] }: Props): React.ReactE
     return null;
   }
 
+  const _onDismiss = (): void => {
+    allst.map(s => s.removeItem());
+    alltx.map(t => t.removeItem());
+  };
+
   return (
     <div className={`ui--Status ${className}`}>
+      {allst.length + alltx.length > 1 && (
+        <div className='dismiss'>
+          <Button
+            isNegative
+            onClick={_onDismiss}
+            label={t('Dismiss all')}
```

Comment:
Would prefer `Dismiss all notifications` (probably moreso with the the icon change indicated.)